pandas/io/formats/format.py

```diff
@@ -172,7 +182,7 @@ def to_string(self):
 
         fmt_values = self._get_formatted_values()
 
-        result = ["{i}".format(i=i) for i in fmt_values]
+        result = ["{i}".format(i=i) for i in fmt_values]  # type: Union[str, List[str]]
```

Comment:
would it be cleaner if this were just `List[str]` and then on 187 use a different variable name?

Reply:
refactored.